runtime/doc/diagnostic.txt

```diff
@@ -499,6 +499,7 @@ Lua module: vim.diagnostic                                    *diagnostic-api*
                           diagnostic instead of prepending it. Overrides the
                           setting from |vim.diagnostic.config()|.
       • {focus_id}?       (`string`)
+      • {border}?         (`string`)
```

Comment:
Can we link to nvim_open_win so that users know what valid values for this option are?

Reply:
before i tried add a see section. but i don't saw it in hover idk.